app/src/main/java/com/example/placefinder/dto/Park.kt

```diff
@@ -2,8 +2,19 @@ package com.example.placefinder.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class Park (@SerializedName("parkCode") var parkCode: String, var parkName: String) {
+/**
+ * A data class representing a Park
+ *
+ * @property parkCode The code of the park
+ * @property parkName The name of the park
+ */
+data class Park (@SerializedName("parkCode") var parkCode: String, @SerializedName("parkName") var parkName: String) {
 
+    /**
+     * Override toString
+     *
+     * @return parkName, parkCode
+     */
```

Comment:
I added Kdoc to better document what the class/methods are doing. Review this link for best practices of Kdocs: https://kotlinlang.org/docs/kotlin-doc.html. This allows for reviewers like me to easily follow your code.

Reply:
See KDoc not above.